packages/gatsby-image/src/index.js

```diff
@@ -48,18 +48,54 @@ const convertProps = props => {
   return convertedProps
 }
 
+/**
+ * Checks if fluid or fixed are art-direction arrays.
+ *
+ * @param currentData  {array}   The props to check for images.
+ * @return {boolean}
```

Comment:
Nit: the output is only a bool if `currentData` is not falsy (like `undefined`). It's probably best to use `!!currentData` in line 59

Reply:
Added : ).